__tests__/src/screens/NationsScreen/index.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import configureStore from 'redux-mock-store';
+
+import { initialState } from '../../../../src/reducers/nations';
+import NationsScreen from '../../../../src/screens/NationsScreen';
+
+test('NationsScreen renders correctly', () => {
+  const initialStateMock = {
+    key: initialState,
```

Comment:
Please pass state with correct key, it should be `nations: initialState`.

Reply:
Done